### DISCUSSION TITLE
Give me Suggestions!

### DISCUSSION THREAD

**Discussion by SirSquidly** [SirSquidly/Oceanic-Expanse#51] [Ideas]
I've been working on an update for the mod, and am interested in hearing what content people would like in the mod.
Any new blocks?
Any Mobs?
Any Config options?
Any other 1.13 backported content?

This is to be left as a general open page for suggestions, so please, fire away!

**Comment by Foreck1** [comment]
Hmm, the mod made by Bluefire and me, [Fishes! Undead, Rise Ye](https://legacy.curseforge.com/minecraft/mc-mods/fishes-undead-rise-ye), adds a so-called guulfish ([render](https://cdn.discordapp.com/attachments/923721813073207306/1162611202170568823/guulfish_tackle.gif?ex=65a2148e&is=658f9f8e&hm=1815644c13c2f49b63e2ad17a3614911f2461a7c70f0936a47938c890a5902b8&)), which is an undead variant of a non-existent fish. 
Perhaps you would be interested in a sort of collaboration and like to add the living version for which I could give provide the texture, model, and geckolib animations? ([render](https://media.discordapp.net/attachments/923721813073207306/1162574617714642974/image.png?ex=65a1f27c&is=658f7d7c&hm=0903d5a935f45baec2a9fe4d435f3dff3ed4d7087116f3c982c8302e94b98c30&=&format=webp&quality=lossless)). The palette is in turn based on the mobs from Depths of Madness, so I'm sort of suggesting adding content that plays on all three 1.12 ocean mods to play on that.
Maybe you don't want the guulfish, but you do want to take inspiration from those other 2 mods to add to your own. I consider this sort of interplay to build something bigger, while expanding on each other's content something really cool.

**Reply by SirSquidly** [reply]
The idea is neat, although I'm not sure it fits the vibe of Oceanic Expanse, and I don't think I want to make my mod require Geckolib just for 1 mob.

Mod cross-pollination is a pretty neat idea though, maybe we could discuss it?

**Comment by Sethgno** [comment]
how about underwater caves with "mineral deposit" blocks inside that can drop ores
![2024-01-06_11 39 38](https://github.com/SirSquidly/Oceanic-Expanse/assets/131620592/03a50fca-ca9c-4ad7-9cb2-b85e8c4be552)
or i could see adding salt deposits that would then craft into chlorine.
and then i can activate my trap card of swapping its drops for harvestcraft salt :3

**Reply by SirSquidly** [reply]
Funnily enough, this is pretty close to two ideas I already plan on implementing!

First off, I am planning on adding deep sea Thermal Vents, that come with rich mineral deposits as a way to give players a risky way to get ore.

I am also planning on revamping Chlorine and incorporating it into the base mod, in the next direct version. I'm going to reskin it as a prismarine-mulch item, as it adds more uses for Prismarine and matches the unnatural and hostile vibe of the Monument

**Comment by Sethgno** [comment]
i just noticed that the "warm ocean" doesnt actually a biome ID and wondering if it can be remedied or would it require making a new biome? do you have plans of adding this as a feature?

**Reply by SirSquidly** [reply]
To be honest, I very much want to, but that is currently outside of my skill level. I very much wish I could.

**Comment by Foreck1** [comment]
What about adding shrimp (or other similar, but not humanoid) "pirates"? A whimsical anthropomorphization of sea life makes for original storytelling. You can base their equipment and weapons around their species for unique inspiration for yourself; maybe their ships are made of coquina. That sort of stuff.

**Reply by SirSquidly** [reply]
Neat suggestion, and I've considered Pirates multiple times during development, though I would probably implement them as an Illager variant based of the Pillager concept texture, or as a Skeleton.

Also, while I'm not sure about adding normal Shrimp, I _really_ want to add a Mantis Shrimp later on. I love Mantis Shrimp.

**Comment by inquist0r1** [comment]
This may be something you have mentioned. But wouldn't jellyfish be a fun, easy addition? You could probably just retexture the baby squid model to be a more pink or blueish color, with maybe a slime-like opaqueness? A good way to make them "sting" without making them hostile is to give them an effect similar to what happens when you stand on a cactus. Maybe a giant jellyfish could be a mini-"boss" that will actively hunt you underwater, but is limited by its giant size. I would recommend making these only spawn in deep ocean, though.

**Reply by SirSquidly** [reply]
Jellyfish are planned for further down the road, when I can get a few concrete utilities/uses for them.

Consider them a confirmed feature.

Not sure about adding a hostile Jellyfish though, as they already would be a defensive mob, and I already have plans for more Hostile ocean mobs, and more completely original hostile ocean mobs.
